docs/usage/docker-shim-removal.md

```diff
@@ -25,9 +25,9 @@ As a shoot owner:
 - if you find dependencies to `docker`, set `.spec.provider.workers[].cri.name: docker` explicitly to avoid defaulting to `containerd` once you update your Shoot to kubernetes v1.22
 
 ## Timeline
-  - **2021-12-30:** Kubernetes v1.21 goes out of maintenance. This is the last version not affected by these changes. Make sure you have tested thoroughly and set the correct configuration for your Shoots!
   - **2021-08-04:** Kubernetes v1.22 released. Shoots using this version get `containerd` as default container runtime. Shoots can still select `docker` explicitly if needed.
   - **2021-12-07:** Kubernetes v1.23 released. Shoots using this version can no longer select `docker` as container runtime.
+  - **2021-12-30:** Kubernetes v1.21 goes out of maintenance. This is the last version not affected by these changes. Make sure you have tested thoroughly and set the correct configuration for your Shoots!
```

Comment:
According to https://kubernetes.io/releases/patch-releases/#1-21 the EoL of v1.21 is `2022-06-28`.

> End of Life for `1.21` is `2022-06-28`

Reply:
Nice catch. The date for 1.22 is also wrong
> End of Life for 1.22 is 2022-10-28

Reply:
Ah, good catch! Seems like I got the date for 1.20 in here. The general information about 1.21 has April 30th as EOL, so I guess I just make it 2022-04 and maybe remove concrete days everywhere?

Reply:
> The general information about 1.21 has April 30th as EOL

Do you mean the statement `1.21 enters maintenance mode on 2022-04-28`? If yes, then `enters maintenance mode` is something different from `reaches end of life` - you currently use `goes out of maintenance` and this means that we should pick the date that is for `reaches end of life`. If you want to use the date that points to `enters maintenance mode`, then you have to rephrase the `goes out of maintenance` part.
If you prefer to use the date for `enters maintenance mode`, then let's use it everywhere for consistency reasons.

> so I guess I just make it 2022-04 and maybe remove concrete days everywhere?

If we have concrete dates for each timeline item, then I would vote to use them to prevent any misunderstanding. Of course they can change with several days or week(s) but this should be fine as our goal is to provide rough timelines.

Reply:
done

Reply:
Ah, I now get what do you mean by 

> The general information about 1.21 has April 30th as EOL

I created https://github.com/kubernetes/website/pull/29443 to tackle the issue on k/website.